docs/application/tizen-studio/release-notes/4-6-release-notes.md

```diff
@@ -0,0 +1,125 @@
+# Tizen Studio 4.6 Release Notes
+
+- Release Date: Mar 31, 2022
+
+## IDE and Tools
+
+### New Feature
+
+- macos Monterey support
+  - Tizen IDE is now supported on macos Monterey
+  - Visual Studio Code Extension for Tizen is now supported on macos Monterey
+- Multi App and Hybrid App support
+  - Added the following Multi app and Hybrid App support in CLI(TZ), VS and VSCode extension for Tizen
+
+  **Multi App**
```

Comment:
Done

Reply:
Line 15: I noticed that this part is indented in comparison with lines 29, 42, and 50. So by making it un-indented, it will be in alignment with the other lines. Please address this issue. 
```suggestion
**Multi App**
```